R/resolve.R

```diff
@@ -152,7 +152,7 @@ resolve_delayed_expr <- function(x, ds, is_value_choices) {
 
   # check returned value
   if (is_value_choices) {
-    if (!is.atomic(res) || anyDuplicated(res)) {
+    if (!isTRUE(checkmate::check_atomic(res)) || anyDuplicated(res)) {
```

Comment:
`check_*` returns a string if `FALSE`, it doesn't matter here due to `isTRUE`, but it's a bit more accurate.

```suggestion
    if (!isTRUE(checkmate::test_atomic(res)) || anyDuplicated(res)) {
```

Reply:
Thanks! I didn't know such a function existed.